src/coreclr/jit/lsra.cpp

```diff
@@ -994,7 +988,7 @@ void LinearScan::setBlockSequence()
         assert(isBlockVisited(block));
     }
 
-    JITDUMP("LSRA Block Sequence: ");
+    JITDUMP("Final LSRA Block Sequence: \n");
```

Comment:
I think you can simplify the code below by not special-casing `MAX`, since `refCntWtd2str` already will return `MAX` if appropriate.

Reply:
Done.